packages/app/src/components/Settings/TableRow.js

```diff
@@ -33,39 +33,23 @@ const TableRow = props => {
           <FontAwesomeIcon
             onClick={() => onEdit(grantCycle)}
             icon='pencil-alt'
-            className='icon-table'
+            className='icon-table-arrow'
           />
         ) : (
           ''
         )}
       </td>
       <td> {grantCycle.name} </td>
-      <td style={{ width: '250px' }}>
-        {/* number of applications displayed */}
-        {/* {grantCycle.nominations.length} */}
-        <button
-          onClick={() => onResultsClick(grantCycle)}
-          className={
-            grantCycle.nominations.length
-              ? 'icon-table icon-right-arrow'
-              : 'icon-right-arrow-disabled'
-          }
-        >
-          {grantCycle.nominations.length}
-        </button>
+      <td
```

Comment:
Added the click to the number of nominations (instead of a button or an arrow like before)